DISCOVER/08_grants_scholarships.md

```diff
@@ -18,7 +18,7 @@ Registration vs. Travel &amp; Lodging
 - Eligibility Requirements
   - Applicants should be 
     - from a traditionally underrepresented and/or marginalized group in the technology and/or open source communities including, but not limited to: persons identifying as LGBTQ, indigenous, women, persons of color, and/or persons with disabilities
-    - and be unable to attend without some financial assistance. (language adapted from [The Linux Foundation](https://www.linuxfoundation.org/event/api-strategy-practice-conference-2017/diversity-scholarship/))
+    - and be unable to attend without some financial assistance. (language adapted from  [The Linux Foundation](https://www.linuxfoundation.org/event/api-strategy-practice-conference-2017/diversity-scholarship/))
```

Comment:
unsubstantial change, can we remove it?

Also if we are adding the same as a footnote in minimal_measures.md maybe do it here as well?